.gitignore

```diff
@@ -400,3 +400,4 @@ FodyWeavers.xsd
 /__azurite_db_blob__.json
 /nuget-push.bat
 **/launchSettings.json
+/site
```

Comment:
@naz-hage we can remove this from `.gitignore`. Instead of using the default `site` folder for the documentation build locally, we can specify that with this command:
```
mkdocs build -d <site_dir>
```

Reply:
@ahmdhjj I like this idea.  How about mkdocs -d %USERPROFILE%\ntools?